src/main/java/com/Insurance/InsTest/Controller/CommonController.java

```diff
@@ -0,0 +1,200 @@
+import com.Insurance.InsTest.Model.DocumentType;
+import com.Insurance.InsTest.Model.Master;
+import com.Insurance.InsTest.Model.PlanType;
+import com.Insurance.InsTest.Model.States;
+import com.Insurance.InsTest.Repository.AccountTypeRepository;
+import com.Insurance.InsTest.Repository.BankRepository;
+import com.Insurance.InsTest.Repository.CountryRepository;
+import com.Insurance.InsTest.Repository.DocumentTypeRepository;
+import com.Insurance.InsTest.Repository.PlanTypeRepository;
+import com.Insurance.InsTest.Repository.StatesRepository;
+import com.Insurance.InsTest.Service.AccountTypeService;
+import com.Insurance.InsTest.Service.BankService;
+import com.Insurance.InsTest.Service.CountryService;
+import com.Insurance.InsTest.Service.DocumentTypeService;
+import com.Insurance.InsTest.Service.PlanTypeService;
+import com.Insurance.InsTest.Service.StatesService;
+
+@RestController
+@RequestMapping("/common")    
+public class CommonController {
+
+	//objects
+	
+	@Autowired
+	 CountryRepository countryRepository;
+	
+	@Autowired
+	 StatesService stateservice;
+	
+	@Autowired
+	 CountryService countryService;
+
+	@Autowired
+	 DocumentTypeService documentTypeService;
+	
+	@Autowired
+	 AccountTypeService accountTypeService;
+	
+	@Autowired
+	 BankService bankService;
+	
+	@Autowired
+	 PlanTypeService typeService; 
+	
+	
+	
+	
+	//methods to get all details of entity
+	
+	@GetMapping("/allcountry")  //ALL
+	public List<Country> getAllCountry()
+	{	
+		try
+		{
+			long totalcount=countryRepository.count();
+			return countryRepository.findAll();
+		}
+		catch (Exception e) {
+			System.out.println(e.getMessage());
+			throw e;
+		}
+	}
+
+
+	@GetMapping("/allstates")  //ALL
+	public List<States> getAllStates()
+	{
+		return stateservice.findAllStates();
+	}
+	
+	@GetMapping("/alldocument")  //ALL
+	public List<DocumentType> getAllDocument()
+	{
+		return documentTypeService.findAllDocumentType();
+	}
+	
+	@GetMapping("/allaccountype")  //ALL
+	public List<AccountType> getAllAccountType()
+	{
+		return accountTypeService.getAllAccountType();
+	}
+		
+	@GetMapping("/allbank")  //ALL
+	public List<Bank> getAllBank()
+	{
+		return bankService.getAllBank();
+	}	
+	
+	
+	@GetMapping("/allplantype")  //ALL
+	public List<PlanType> getAllPlanType()
+	{
+		return typeService.getAllPlanType();
+	}
+		
+	
+	
+	//method to get alldetails of the present entities
+	
+	@GetMapping("/getmasterdata")
+	public Master getAllEntity() throws ResourceNotFoundException
+	{
+		Master master=new Master();
+	
+		master.country=getAllCountry();
+		
+		master.state=getAllStates();
+		
+		master.documenttype=getAllDocument();
+		
+		master.accounttypes=getAllAccountType();
+		
+		master.bank=getAllBank();
+		
+		master.plantype=getAllPlanType();
+		
+		return master;
+				
+	}
+	
+	//method to get by id
+	
+	@GetMapping("/countryid/{id}")//byid
+	public ResponseEntity<Country> getCountryById(@PathVariable(value="id") Long countryId )
```

Comment:
here handle the exception for **getmasterdata** api and also handle for all the api except allcountry(api)